src/ADP.Portal.Api/Services/AdoRestAPIService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using ADP.Portal.Api.Config;
+using ADP.Portal.Api.Models.Ado;
+using ADP.Portal.Core.Ado.Infrastructure;
+using ADP.Portal.Core.Helpers;
+using Microsoft.Extensions.Options;
+using System.Net.Http.Headers;
+
+namespace ADP.Portal.Api.Services
+{
+    public class AdoRestApiService : IAdoRestApiService
+    {
+        private readonly ILogger<AdoRestApiService> logger;
+        private readonly string adoOrgUrl;
+        private readonly HttpClient client;
+
+        public AdoRestApiService(ILogger<AdoRestApiService> logger, IOptions<AdoConfig> configuration)
+        {
+            this.logger = logger;
+            adoOrgUrl = configuration.Value.OrganizationUrl;
+            client = new HttpClient();
+            client.DefaultRequestHeaders.Accept.Add(
+                    new MediaTypeWithQualityHeaderValue("application/json"));
+
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic",
+                Convert.ToBase64String(
+                    System.Text.Encoding.ASCII.GetBytes(
+                        string.Format("{0}:{1}", "", configuration.Value.PatToken))));
```

Comment:
{0} is always an empty string here, is that intentional?
Maybe using an interpolated string would be better?
`$":{configuration.Value.PatToken}"`

Reply:
Yes it is intentional. Thats how the auth is constructed for ADO

Reply:
to simply, we can change it to interpolated string  $":{configuration.Value.PatToken}" will produce the same output as the string format.